lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -192,9 +192,9 @@ describe('ReportUIFeatures', () => {
         expect(lhr.audits['unused-javascript'].details.headings).toHaveLength(0);
         lhr.audits['unused-javascript'].details.headings = [
           /* t-disable max-len */
```

Comment:
seems like these are not necessary anymore and not even valid 😆

Reply:
lol